client/src/components/History/CurrentHistory/HistoryDetails.vue

```diff
@@ -15,6 +15,7 @@
 <script>
 import short from "components/directives/v-short";
 import Details from "components/History/Layout/Details";
+import store from "store/index";
```

Comment:
All vue components will have the store included; no need to import here, you can just use `this.$store` below.